src/modules/uavcan/actuators/esc.cpp

```diff
@@ -49,12 +49,24 @@ UavcanEscController::UavcanEscController(uavcan::INode &node) :
 	_uavcan_sub_status(node),
 	_orb_timer(node)
 {
+	if (_perfcnt_invalid_input == nullptr) {
+		errx(1, "uavcan: couldn't allocate _perfcnt_invalid_input");
+	}
+
+	if (_perfcnt_scaling_error == nullptr) {
+		errx(1, "uavcan: couldn't allocate _perfcnt_scaling_error");
+	}
+
+	if (_perfcnt_broadcast_elapsed == nullptr) {
+		errx(1, "uavcan: couldn't allocate _perfcnt_broadcast_elapsed");
+	}
 }
 
 UavcanEscController::~UavcanEscController()
 {
 	perf_free(_perfcnt_invalid_input);
 	perf_free(_perfcnt_scaling_error);
+	perf_free(_perfcnt_broadcast_elapsed);
```

Comment:
On a full system you can't fit them on a screen and their strings eat flash space. I'm trying to also remove the boiler-plate shell handing and error handling code. That should allow us to recuperate a reasonable amount of space, allowing us to live well in our 1M prison (thanks ST!).